### DISCUSSION TITLE
Ability to future workouts and let the app remind them on that particular day

### DISCUSSION THREAD

**Discussion by unfazedEgnimadevspace** [unfazedEgnimadevspace/Gracker#54] [Ideas]
Hello @americodls 

What do you think about this feature, Users can log into the application just to plan future workouts on specific dates and have the app alerts them through email a day before and on that particular day also so they can plan adequately and successfully complete that exercise.

They model(haven't decided a name yet) will also have a completed attribute that will be initially false, and only becomes true when the user successfully completes the exercise.

What do you think?

**Comment by americodls** [comment]
Sounds good!
Maybe you should start thinking about the UI.
Start by designing the experience you want the user to have.
Probabaly with that ready you will be able to find the names of models and attributes.
